### PR TITLE
Refactor for wso2-extension to siddhi-io repo move

### PR DESCRIPTION
**Pull request by ajkarki**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/docs/api/2.0.1-SNAPSHOT.md [hunk 1]
```diff
@@ -0,0 +1,1876 @@
+# API Docs - v2.0.1-SNAPSHOT
```
Comment (mohanvive) [inline component/docs/api/2.0.1-SNAPSHOT.md]: Please revert this file addition.

File: component/docs/api/latest.md [hunk 2]
```diff
@@ -0,0 +1,1876 @@
+# API Docs - v2.0.1-SNAPSHOT
```
Comment (mohanvive) [inline component/docs/api/latest.md]: Please revert this file change. Don't include changes to MD files in ur commit.

File: component/docs/api/2.0.1-SNAPSHOT.md (hunk 1 above)
Comment (ajkarki) [reply component/docs/api/2.0.1-SNAPSHOT.md]: Fix has been provided.

File: component/docs/api/latest.md (hunk 2 above)
Comment (ajkarki) [reply component/docs/api/latest.md]: Fix has been provided.

**Comment by mohanvive** [comment]
@ajkarki I don't see that you have revered the changes. Please revert the doc changes.

**Pull request merged**